cerberus-dashboard/package.json

```diff
@@ -12,7 +12,8 @@
     "start": "react-scripts start",
     "build": "react-scripts build && rm -rf dist && mv build dist",
     "test": "react-scripts test --env=jsdom",
-    "logs": "multitail logs/*"
+    "logs": "multitail logs/*",
+    "preinstall": "npx npm-force-resolutions"
```

Comment:
what's this?

Reply:
npm-force-resolutions is an npm plugin that forces your node dependencies to a specific version. 

In this case, react-scripts@3.4.1 (the most current version of react-scripts) pulls in yargs-parser 11.1.1 which has a high severity CVE. We need to use npm-force-resolutions to force yargs-parser to 13.1.2 where the vulnerability has been resolved. Once react-scripts updates their version of yargs-parser we can unforce this. 

Here are the package details. https://github.com/rogeriochaves/npm-force-resolutions